heron/executor/src/python/heron_executor.py

```diff
@@ -677,7 +677,7 @@ def _untar_if_needed(self):
     if self.pkg_type == "tar":
       os.system("tar -xvf %s" % self.topology_bin_file)
     elif self.pkg_type == "pex":
-      os.system("unzip %s" % self.topology_bin_file)
+      os.system("unzip -qq -n %s" % self.topology_bin_file)
```

Comment:
The thing is that when launching we need to clean the directory. I've updated the pr to ensure that we clean the working directory while setting it up. This way, new topology launches will be ok.